di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/config/OrchestratorConfig.java

```diff
@@ -47,6 +47,16 @@ public class OrchestratorConfig {
     public static final String BASIC_AUTH_PASSWORD =
             getConfigValue("ORCHESTRATOR_BASIC_AUTH_PASSWORD", null);
 
+    public static final String INHERITED_IDENTITY_JWT_SIGNING_KEY =
+            getConfigValue("INHERITED_IDENTITY_JWT_SIGNING_KEY", null);
+    public static final String INHERITED_IDENTITY_JWT_ISSUER =
+            getConfigValue(
+                    "INHERITED_IDENTITY_JWT_ISSUER", "<https://oidc.hmrc.gov.uk/migration/v1>");
+    public static final String INHERITED_IDENTITY_JWT_VTM =
+            getConfigValue("INHERITED_IDENTITY_JWT_VTM", "<https://hmrc.gov.uk/trustmark>");
```

Comment:
created a JWT based on the example provided in the task. Could you please add an example JWT with correct values for all fields to the task? The reason I requested a JWT containing all fields was because of our previous discussion on the Slack channel.

Reply:
I don't think there's a complete example of a 'valid' VC yet, because a few of the details are TBD, and the examples in the RFC are for prod (not the stub). I think it should be roughly:
```
{
  "sub": "urn:fdc:gov.uk:2022:A_w9_Or3gyI-ygrzH7cNZe9qTYFkUOzHF-KoPWmb9QY",
  "iss": "https://orch.stubs.account.gov.uk/migration/v1",
  "jti": "urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6",
  "nbf": 1541493724,
  "vot": "CL200",
  "vtm": "https://hmrc.gov.uk/trustmark",
  "vc": {
    "type": [
      "VerifiableCredential",
      "IdentityCheckCredential"
    ],
    "credentialSubject": {
      "name": [
        {
          "nameParts": [
            {
              "value": "Alice",
              "type": "GivenName"
            },
            {
              "value": "Jane",
              "type": "GivenName"
            },
            {
              "value": "Laura",
              "type": "GivenName"
            },
            {
              "value": "Doe",
              "type": "FamilyName"
            }
          ]
        }
      ],
      "birthDate": [
        {
          "value": "1970-01-01"
        }
      ],
      "socialSecurityRecord": [
        {
          "personalNumber": "AB123456C"
        }
      ]
    }
  },
  "evidence": [
    {
      "type": "IdentityCheck",
      "txn": "d22f8cb0-82f2-4526-9d88-07ccd358ab13",
      "strengthScore": 2,
      "validityScore": 2,
      "verificationScore": 1
      "checkDetails": [
        {
          "checkMethod": "data"
        }
      ]
    }
  ]
}
```
This is based on the draft RFC (https://github.com/govuk-one-login/architecture/pull/399/files), adding the JTI and updating the issuer to reflect that it's from a stub.

Reply:
Thank you.

Reply:
@rpayal good question

We don't expect to receive an address in the inherited identity VC - and as you've spotted, this does mean we won't return an address in the user identity when returning an operation profile.

I haven't explicitly considered the impact of this - if this is going to break something we might need to soften our handling of a missing address.

Reply:
@Joe-Edwards-GDS Currently ProvenUserIdentity take user to journey error path if address is not found.

Reply:
Ok thanks - we probably need a new ticket for this (unless you want to add it to PYIC-4339)

Reply:
changed iss value with 'https://orch.stubs.account.gov.uk/migration/v1' according above jwt example. But, can be updated by modifying the parameter INHERITED_IDENTITY_JWT_ISSUER in the configuration.